app/js/lib/helpers.js

```diff
@@ -49,3 +49,32 @@ export class ManifestHelper {
       (manifest1.type === 'privileged' && manifest2.type !== 'certified');
   }
 }
+
+export class ActivityHandler {
```

Comment:
nit: let's call this `ActivityHelper` to be more consistent with the other classes here.